### DISCUSSION TITLE
cshrc error

### DISCUSSION THREAD

**Discussion by Abdallahbalushi** [gmtsar/gmtsar#850] [Q&A]
When i am trying to run prep_data_linux.csh command, it opens the .cshrc file with the wget sentence highlighted in red and it cannot be adjusted or even closed. 

Any ideas on how to fix this? 
![image](https://github.com/gmtsar/gmtsar/assets/150544398/249fc601-346d-4237-9670-8e9834e8b439)
![image](https://github.com/gmtsar/gmtsar/assets/150544398/59b923f0-0263-4c54-8c95-2aa40c8d6a39)

**Comment by Xiaohua-Eric-Xu** [comment]
I think you should not be running `vi` in your `~/.tcshrc` file. Just adding the alias should work.

**Reply by Abdallahbalushi** [reply]
Hi Eric, 
I removed the vi ~/.cshrc and leave the alias wgetasf. However, when I tried to run prep_data_linux.csh, it says that it's unmatched. Please see the image below. The problem? 
![image](https://github.com/gmtsar/gmtsar/assets/150544398/ce21e8e8-0d9f-4c2b-9e25-b46d11fd3003)

**Reply by Xiaohua-Eric-Xu** [reply]
Could it be that in your script you somehow did not pair a quote mark? e.g., using `'` and `‘` in your alias?
